src/pages/landing-page/LandingPage.jsx

```diff
@@ -39,17 +38,29 @@ const LandingPage = () => {
                             cotagory to explore more.
                         </p>
                         <div className="btn-container">
-                            <Link className="btn btn-primary" to="/products">Women's Wear</Link>
-                            <Link className="btn btn-secondary" to="/products">Men's Wear</Link>
+                            <Link className="btn btn-primary" to="/products">Silk Special</Link>
+                            <Link className="btn btn-secondary" to="/products">Wedding Special</Link>
                         </div>
                     </div>
                 </section>
             </main>
+            <section className="featured-section">
+            <h3 className="heading heading-avatar heading-card heading-ecom"> </h3>
+                <div className="featured-product">
+                    <img className="responsive" src={Ban2} alt="error" />
```

Comment:
didnt understand Ban2, you can write in discription

Reply:
Its banner . I saw most of the sites using this convention for banners so did it.